freqtrade/rpc/rpc.py

```diff
@@ -576,6 +596,9 @@ def _rpc_trade_statistics(
             'winning_trades': winning_trades,
             'losing_trades': losing_trades,
             'profit_factor': profit_factor,
+            'winrate': winrate,
+            'expectancy': expectancy,
+            'expectancy_rate': expectancy_rate,
```

Comment:
if you intend to expose these through the API, too (which we should) - you'll need to add them to api_schemas for the corresponding endpoint.

Reply:
Ah I knew I forgot something!

Reply:
be sure to double-check api tests - not sure how this endpoint is tested ... but the corresponding one might fail after this.

Reply:
I swear if I get another trailing whitespace precommit I'll explode.

Reply:
fix your editor? trailing commits can be removed by vscode automatically ... 
just make sure to disable it for *.md - 2 trailing spaces there are a paragraph :laughing: